libnavigator/src/main/java/com/mapbox/navigation/navigator/internal/MapboxNativeNavigator.kt

```diff
@@ -76,21 +75,9 @@ interface MapboxNativeNavigator {
      */
     fun updateSensorData(sensorData: SensorData): Boolean
 
-    /**
-     * Gets the status as an offset in time from the last fixed location. This
-     * allows the caller to get predicted statuses in the future along the route if
-     * the device is unable to get fixed locations. Poor reception would be one reason.
-     *
-     * This method uses previous fixes to snap the user's location to the route
-     * and verify that the user is still on the route. This method also determines
-     * if an instruction needs to be called out for the user.
-     *
-     * @param navigatorPredictionMillis millis for navigation status predictions.
```

Comment:
Same as above, the prediction value should now be passed to `PollingConfig`.

Reply:
This is important to address so that we can effectively support use-cases with higher frequency signal than 1Hz.

Reply:
Added `InternalUtils` https://github.com/mapbox/mapbox-navigation-android/blob/b2ad867af1d4f2dd4851618c354b7ebcaa3a3433/libnavigation-core/src/main/java/com/mapbox/navigation/core/internal/utils/InternalUtils.kt Please let me know what you think 🙏

Reply:
cc @DmitryAk

Reply:
@Guardiola31337 @LukasPaczos please see my comment https://github.com/mapbox/mapbox-navigation-android/pull/4419/files#r655903753
It looks like previously `InternalUtils` methods accepted values in milliseconds, now they accept values in seconds. It's important to not miss this, so the `InternalUtils` consumers behavior will not be broken.